packages/react/src/components/SidePanel/_side-panel.scss

```diff
@@ -16,105 +16,80 @@ $border-width: 1px;
     left $duration--moderate-02, right $duration--moderate-02;
   transition-timing-function: carbon--motion(standard);
 
-  &.#{$iot-prefix}--side-panel__right {
+  &--end {
     right: 0;
   }
 
-  .panel {
-    height: 100%;
-
-    .panel-content-wrapper {
-      display: flex;
-      flex-direction: column;
-      height: 100%;
-    }
-
-    .panel-content {
-      padding-bottom: $spacing-09;
-    }
-  }
-
-  .close-button {
-    position: absolute;
+  &--close-button {
+    position: absolute !important;
```

Comment:
Instead of using `!important` we should try to increase the specificity of this selector.